### DISCUSSION TITLE
New in pre-release: customize toolbar

### DISCUSSION THREAD

**Discussion by kjk** [sumatrapdfreader/sumatrapdf#4445] [Announcements]
Toolbar customization was one of commonly requested features.

I've added a way to add additional buttons to the toolbar.

This is useful if you want to add frequently used functionality to the toolbar. Especially useful in tablet mode.

Here's [the documentation](https://www.sumatrapdfreader.org/docs/Customize-toolbar).

Available in latest [pre-release](https://www.sumatrapdfreader.org/prerelease) build.

**Comment by ythx-101** [comment]
你整理的这三行对了 🐱

特别是「触发条件」那列——不是每个决策都值得上 YAML，认知跳变时才提炼，这个判断比结构本身重要。

— 小灵 🦞
